test/DynamoCoreTests/CodeBlockNodeTests.cs

```diff
@@ -212,6 +212,127 @@ public void TestStatement_ArrayReference()
 #endif
         protected double tolerance = 1e-4;
 
+        // Note: This test fails, even though the individual items pass as separate tests
```

Comment:
This code is here to show an issue with the testing system, it can be removed if desired, not sure what the policy is for this.

Reply:
what is the failure mode here?

Reply:
The failure mode is that the second test fails even though if the two tests are run in separate test runs they both pass.

Reply:
If you look at TestFunctionDefaultParameters versus the following TestFunctionSingleDefaultParameter and TestFunctionTwoDefaultParameters you can see that the tests are the same, yet if they are concatenated into one test the test will fail. I did not really look into why, I was just notating that there is potentially an issue with the testing system there that may trip up other people in the future.

Reply:
hmm - the difference between running these in a single test and and in multiple tests is that Dynamo is being shut down and new workspaces are being opened in the new tests... that could be a problem with the testing system or it could be a real bug...

Reply:
when you interact manually with Dynamo you cannot reproduce this failure?

Reply:
To memorialize our recent Zoom conversation, each test passes separately, but putting them both in the same test case fails. Put another way, I can reproduce both issues manually, but in the testing framework having both tests run together fails. I would guess at this point that it is an issue with the testing framework and not Dynamo, but this is just a guess at this point. The main thought was just to get it notated so that other people may avoid some future pain, I'm not sure how much of importance this particular issue has for the everyday user.